README.md

```diff
@@ -186,125 +192,128 @@ If you don't have a React application already, start with [create react app](htt
-    // Replace these values
-    const bearerToken = '{REPLACE_ME}'; // retrieved from CP4D Admin UI under instance details which expires daily
-    const url = '{REPLACE_ME}'; // retrieved from CP4D Admin UI under instance details
-    const version = '{REPLACE_ME}'; // YYYY-MM-DD date format
-    const projectId = '{REPLACE_ME}'; // retrieved from Discovery Tooling UI
-    const App = () => {
-      let authenticator, searchClient, success;
-      try {
-        // see https://github.com/IBM/node-sdk-core/blob/master/AUTHENTICATION.md#bearer-token-authentication
-        authenticator = new BearerTokenAuthenticator({ bearerToken });
-        searchClient = new DiscoveryV2({ url, version, authenticator });
-        success = true;
-      } catch (err) {
-        console.error(err);
-      }
-      return success ? (
-          <DiscoverySearch searchClient={searchClient} projectId={projectId}>
-            <SearchInput />
-            <SearchResults />
-            <SearchFacets />
-            <ResultsPagination />
-            <DocumentPreview />
-          </DiscoverySearch>
-      ) : (
-        setupMessage()
-      );
-    };
-    function setupMessage() {
-      return (
-        <div style={{
-          textAlign: 'center',
-          margin: '20%',
-          fontSize: '1.5rem',
-        }}>
-          Please replace the constants in App.js in order to see the Discovery sample application.
-          <br /><br />
-          Check the console log for more information if you have replaced these constants and are still seeing this message.
-        </div>
-      );
-    }
-    export default App;
-  ```
-
-  For more information on how each component can be customized and configured, check out our hosted [storybook](https://watson-developer-cloud.github.io/discovery-components)
-
-  ### Interacting with Discovery data in custom components
-
-  Interacting with Discovery data is facilitated by the use of [React Context](https://reactjs.org/docs/context.html). The only requirement for a component to consume or request data is that it be nested underneath the `DiscoverySearch` component.
-
-  ex.
-
-  ```jsx
-  // src/App.js
-
-  import React from 'react';
-  import { DiscoverySearch } from '@ibm-watson/discovery-react-components';
-  import { MyCustomComponent } from './MyCustomComponent.js';
-
-  const App = () => {
-    // see more detailed searchClient example above for `searchClient` variable
-    return (
-      <DiscoverySearch searchClient={searchClient} projectId={'REPLACE_ME'}>
-        <MyCustomComponent />
-      </DiscoverySearch>
-    );
-  };
+```jsx
+// src/App.js
+import React from 'react';
+import {
+  DiscoverySearch,
+  SearchInput,
+  SearchResults,
+  SearchFacets,
+  ResultsPagination,
+  DocumentPreview
+} from '@ibm-watson/discovery-react-components';
+import DiscoveryV2 from 'ibm-watson/discovery/v2';
+import '@ibm-watson/discovery-styles/scss/index.scss';
+// Make sure to setup your ibm-credentials.env file first
```

Comment:
More details at https://github.com/watson-developer-cloud/node-sdk#authentication but this makes it more flexible than the programmatic approach (like using `BearerTokenAuthenticator` explicitly)

I figured this was less verbose and kept the content related to this repo rather than focusing on the authentication details which is handled by our peer dependencies

Reply:
let's add a comment here that calls out that discoveryv2 reads environment variables for the authenticator